user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations.md

```diff
@@ -8,279 +8,77 @@ Many visualizations are available in the Dashboards app and Low-Code Apps, in th
+| ![Tree](~/user-guide/images/Tree.svg) | A [tree](xref:DashboardTree) allows users to select one or more items in a tree view. |
+| ![Parameter picker](~/user-guide/images/Parameter_Picker.svg) | A [parameter picker](xref:DashboardParameterPicker) allows users to select multiple parameters from a predefined list. |
+| ![Trend statistics](~/user-guide/images/Trend_Statistics.svg) | A [trend statistics](xref:DashboardTrendStatistics) component displays the minimum, average, and maximum value of one or more trended parameters. |
 
 ### Basic controls
 
-<style>
-.row {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 0 4px;
-}
- 
-/* Create five equal columns that sit next to each other */
-.column {
-  flex: 20%;
-  max-width: 20%;
-  padding: 0 11px;
-}
-</style>
-
-<div class="row"> 
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/Dropdown.html" title="DropDown" target="_self"><img src="~/user-guide/images/Dropdown.svg" style="width:100%"></a>
-  </div>
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/DashboardNumericInput.html" title="Numeric Input" target="_self"><img src="~/user-guide/images/Numeric_Input.svg" style="width:100%"></a>
-  </div>
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/DashboardSearchInput.html" title="DropDown" target="_self"><img src="~/user-guide/images/Search_Input.svg" style="width:100%"></a>
-  </div>
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/DashboardTextInput.html" title="Text Input" target="_self"><img src="~/user-guide/images/Text_Input.svg" style="width:100%"></a>
-  </div>  
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/Timerange.html" title="Time Range" target="_self"><img src="~/user-guide/images/Time_Range.svg" style="width:100%"></a>
-  </div>
-</div>
-
-<div class="row"> 
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/Toggle.html" title="Toggle" target="_self"><img src="~/user-guide/images/Toggle.svg" style="width:100%"></a>
-  </div>
-</div>
+| Visualization | Description |
+|:--:|--|
+| ![Dropdown](~/user-guide/images/Dropdown.svg) | A [dropdown](xref:DashboardDropdown) allows users to select an item in a dropdown list. |
+| ![Numeric input](~/user-guide/images/Numeric_Input.svg) | A [numeric input](xref:DashboardNumericInput) component allows users to enter numbers, which will then be available as numeric data in the dashboard or app. |
```

Comment:
"which will then be available as X data in the dashboard or app" is the same for all basic controls so not sure if this needs to be added for every item.